package_manager/contracts/PR/RepoPackage.sol

```diff
@@ -0,0 +1,98 @@
+pragma solidity ^0.4.24;
+
+
+/**
+ * @title RepoPackage
+ */
+contract RepoPackage {
```

Comment:
As a general comment, I feel like the API is a bit convoluted. For instance, are the `getByVersionId` getters needed, or could we remove them? Same for `getLatestForContractAddress`. I know that these methods are in the original AragonPackage, can we reach out to them and check if they do need all of them?

Reply:
Totally agree, the main idea of this PoC was to validate that Aragon's interface could be compatible with ours, we should now work together towards a common and better and minimum interface :)